Parser/StarmonMiniParse.m

```diff
@@ -314,7 +314,7 @@
     iContent = i + nColumnTime;
     
     switch header.param(iParam).column
-        case 'Temp(�C)'
+        case 'Temp0x280xB0C0x29' %degrees C
```

Comment:
How did you get this value? On my machine I get 'Temp0x280xFFFDC0x29' as a translation of 'Temp(°C)'.